Okta.AspNet.Abstractions/OktaWebOptions.cs

```diff
@@ -36,5 +36,10 @@ public class OktaWebOptions
         /// The clock skew.
         /// </value>
         public TimeSpan ClockSkew { get; set; } = DefaultClockSkew;
+
+        /// <summary>
+        /// Gets or sets the uri of your organization's proxy server.  The default is null.
+        /// </summary>
+        public string ProxyAddress { get; set; }
```

Comment:
Is this property also available in `OktaWebApiOptions`? If so, we should consider move this property so it's only available for MVC projects. I think this doesn't make sense in Web API. Thoughts?

Reply:
Why would it not be valid?  Proxy servers can be used anywhere in a request pipeline, seems like developers should be allowed the opportunity to specify it if appropriate for their environment.

Reply:
Looking closer, I think I see your point.  I'll review more closely.